### PR TITLE
2173: Did you mean?

### PR DESCRIPTION
**Pull request by troelslenda**
http://platform.dandigbib.org/issues/2173

Requires approval and merge of #553 in order to work, as it requires the hook `hook_ting_search_results`

This PR also introduces the module `ting_subsearch_common` which is a requirement for other subsearch modules.

### CODE REVIEW

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: cdb41a28dc1c6c491e7cf0ff38152fb583e9181c
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: 6bcbe2f989c36bcc72d031a7da4a67371a9afcd3
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: cdb41a28dc1c6c491e7cf0ff38152fb583e9181c
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: 6bcbe2f989c36bcc72d031a7da4a67371a9afcd3
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: c72b0647a2eaeb7b3435c15e4fe88f6002339cef
  * Your commit message body contains a line that is longer than 72 characters

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: c72b0647a2eaeb7b3435c15e4fe88f6002339cef
  * Your commit message body contains a line that is longer than 72 characters

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

File: modules/ting_subsearch_opensuggestion/ting_subsearch_opensuggestion.module [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+ */
+function ting_subsearch_opensuggestion_libraries_info() {
+  return array(
+    'guzzle' => array(
+      'name' => 'Guzzle',
+      'vendor url' => 'https://github.com/guzzle/guzzle',
+      'download url' => 'https://github.com/guzzle/guzzle',
+      'version' => '6.2',
+      'xautoload' => function ($adapter) {
+        $adapter->composerJson('composer.json');
```
Comment (troelslenda) [inline modules/ting_subsearch_opensuggestion/ting_subsearch_opensuggestion.module]: The issues reported by Scrutinizer regarding indentation isen't in my opinion an issue.
Locally PHP_CodeSniffer reports this as OK. Perhaps Scrutinizer is running an older version of PHP_CodeSniffer ?

File: modules/ting_subsearch_common/ting_subsearch_common.module [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+ */
+function ting_subsearch_common_libraries_info() {
+  return array(
+    'guzzle' => array(
+      'name' => 'Guzzle',
+      'vendor url' => 'https://github.com/guzzle/guzzle',
+      'download url' => 'https://github.com/guzzle/guzzle',
+      'version' => '6.2',
+      'xautoload' => function ($adapter) {
+        $adapter->composerJson('composer.json');
```
Comment (troelslenda) [inline modules/ting_subsearch_common/ting_subsearch_common.module]: The issues reported by Scrutinizer regarding indentation isen't in my opinion an issue.
Locally PHP_CodeSniffer reports this as OK. Perhaps Scrutinizer is running an older version of PHP_CodeSniffer ?

Comment (troelslenda) [inline modules/ting_subsearch_common/ting_subsearch_common.module]: Scrutinizer now reports 167 issues. Of these 163 issues were fixed. The 4 issues regarding indentation are still there.

**Pull request closed without merging**